source/introduction.md

```diff
@@ -93,17 +111,21 @@ this_group:
   - key3: value3
 ```
 The first represents an unordered dictionary of key, value pairs and the second represents an ordered 
-dictionary.
+dictionary of key value pairs.
```

Comment:
The writing of "key, value pairs" then "key value pairs" is inconsistent and a bit confusing, grammatically, when reading at first. May be write, e.g., "{key, value} pairs" or "[key, value] pairs"?

Reply:
Now using dash.